### PR TITLE
ADEN-4330 VUAP tests

### PR DESCRIPTION
**Pull request by v3rth**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/VUAP.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        return iframe;
+    }
+
+    public void play() {
+        driver.switchTo().frame(iframe);
+
+        try {
+            Thread.sleep(2000); // TODO replace with "wait for video will be ready to play"
+        } catch (InterruptedException e) {
+            e.printStackTrace();
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/VUAP.java]: CRITICAL Use a logger to log this exception. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1148)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues:
* CRITICAL 1 critical
* MINOR 1 minor
* INFO 1 info

Watch the comments in this conversation to review them.

**Comment by bulaszenko** [comment]
LGTM

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestVUAP.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+public class TestVUAP extends TemplateNoFirstLoad {
+  private static final Dimension DESKTOP_SIZE = new Dimension(1920, 1080);
+
+  @Test(
+          groups = "AdsUapDesktop"
+  )
+  public void testVideoPlayFlow() {
+    Page page = new Page("project43", "SyntheticTests/VUAP");
+    AdsBaseObject ads = new AdsBaseObject(driver, urlBuilder.getUrlForPage(page), DESKTOP_SIZE);
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestVUAP.java]: MAJOR Remove this unused "ads" local variable. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1481)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Pull request merged**